azext_edge/edge/providers/check/base/deployment.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
```

Comment:
I'm thinking of the naming of the new files under this base folder... pods.py, nodes.py are plurals, therefore should we keep consistent to have plural on deployment.py and display.py as well?🤔

Reply:
well not sure if displays and deployments would make sense here since the idea was that there are multiple pods, (possibly) nodes, resources, etc while one (AIO) deployment, one display, one check manager

Reply:
I will just make everything singular